cmd/run.go

```diff
@@ -82,12 +82,28 @@ func SetupRunFlags(cmd *cobra.Command) {
 		- hypersync: Will sync by downloading historical state, and will NOT
 		  download historical blocks. Can only be set if HyperSync is true.`)
 
+	// PoS Validator
+	cmd.PersistentFlags().String("pos-validator-seed", "", "A BIP39 seed phrase used to generate the private key of the "+
+		"Proof of Stake validator. Setting this flag automatically makes the node run as a Proof of Stake Validator.")
+	cmd.PersistentFlags().Uint64("pos-block-production-interval-milliseconds", 1500, "The interval in milliseconds "+
+		"between blocks produced by the Proof of Stake block proposer. The default value is 1.5 seconds, or 1500 milliseconds.")
+	cmd.PersistentFlags().Uint64("pos-timeout-base-duration-milliseconds", 30000, "The base duration in milliseconds "+
+		"that the Proof of Stake block proposer waits before timing out and producing a block. The default value is "+
+		"30 seconds, or 30000 milliseconds.")
+
 	// Mempool
 	cmd.PersistentFlags().Uint64("mempool-backup-time-millis", 30000,
 		"The frequency in milliseconds with which the mempool will persist its state to disk. "+
 			"The default value is 30 seconds, or 30,000 milliseconds.")
 	cmd.PersistentFlags().Uint64("max-mempool-pos-size-bytes", 3000000000,
 		"The maximum size of the PoS mempool in bytes. The default value is 3GB.")
+	cmd.PersistentFlags().Uint64("mempool-fee-estimator-num-mempool-blocks", 1,
```

Comment:
These mempool params seem appropriate as flags. In my view, any consensus variables that require coordination among node operators should be in ParamUpdater probably. Anything that individual nodes can update independently should be flags.

Reply:
Sounds good. Will move these to ParamUpdater too